sourced/ml/utils/engine.py

```diff
@@ -36,10 +36,12 @@ def add_engine_args(my_parser, default_packages=None):
     my_parser.add_argument(
         "--engine", default=EngineDefault.VERSION,
         help="source{d} engine version.")
-    my_parser.add_argument("--repository-format", default="siva",
-                           help="Repository storage input format.")
-    my_parser.add_argument("--explain", action="store_true",
-                           help="Print the PySpark execution plans.")
+    my_parser.add_argument(
```

Comment:
The lines perfectly fit 100 chars limit, please revert this change.

Reply:
it is for equability. Do you agree that 
```python
    my_parser.add_argument(
        "--bblfsh", default=EngineDefault.BBLFSH,
        help="Babelfish server's address.")
    my_parser.add_argument(
        "--engine", default=EngineDefault.VERSION,
        help="source{d} engine version.")
    my_parser.add_argument(
        "--repository-format", default="siva",
        help="Repository storage input format.")
    my_parser.add_argument(
        "--explain", action="store_true",
        help="Print the PySpark execution plans.")
```
is better than
```python
    my_parser.add_argument(
        "--bblfsh", default=EngineDefault.BBLFSH,
        help="Babelfish server's address.")
    my_parser.add_argument(
        "--engine", default=EngineDefault.VERSION,
        help="source{d} engine version.")
    my_parser.add_argument("--repository-format", default="siva",
                           help="Repository storage input format.")
    my_parser.add_argument("--explain", action="store_true",
                           help="Print the PySpark execution plans.")
```

Or I can move everything into
```python
    my_parser.add_argument("--bblfsh", default=EngineDefault.BBLFSH,
                           help="Babelfish server's address.")
    my_parser.add_argument("--engine", default=EngineDefault.VERSION,
                           help="source{d} engine version.")
    my_parser.add_argument("--repository-format", default="siva",
                           help="Repository storage input format.")
    my_parser.add_argument("--explain", action="store_true",
                           help="Print the PySpark execution plans.")
```

Reply:
Let's choose the second option - it saves 4 lines of the precious vertical space.